src/doc_examples/tailwind.rs

```diff
@@ -1,4 +1,12 @@
 use dioxus::prelude::*;
 
-// Urls are relative to your Cargo.toml file
-const TAILWIND_URL: Asset = asset!("/assets/tailwind.css");
+#[component]
+fn app() -> Element {
+    rsx! {
+        // The style component inserts a style link into the head of the document
+        document::Style {
```

Comment:
Here it must be Stylesheet https://dioxuslabs.com/blog/release-060#document-elements